src/main/groovy/edu/oregonstate/mist/restpr/api/User.groovy

```diff
@@ -0,0 +1,26 @@
+package edu.oregonstate.mist.restpr.api
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+import javax.xml.bind.annotation.XmlRootElement
+
+/**
+ * Created by georgecrary on 6/26/15.
+ */
+
+@XmlRootElement(name = "user")
```

Comment:
if the `@XmlRootElement` annotation is necessary, explain why. otherwise delete it.